python/Ganga/Runtime/bootstrap.py

```diff
@@ -37,6 +37,17 @@
 else:
     _gangaVersion = "SVN_TRUNK"
 
+
+def new_version_format_to_old(version):
+    """
+    Convert from 'x.y.z'-style format to 'Ganga-x-y-z'
+
+    Example:
+        >>> new_version_format_to_old('Ganga-6-1-11')
+        '6.1.11'
+    """
+    return 'Ganga-'+version.replace('.', '')
```

Comment:
I think that there might be a couple of problem in this function as is:
* The doctest seems to be converting old to new which is the opposite of the function doc and name. In addition, the output expected seems to be missing the Ganga- prefix which the return line suggests should be present.
* I think you meant to have the replace char as a -, i.e.
```python
version.replace('.', '-')
```
* As you don't strip Ganga- from the `version` argument, the return would in the case of your doctest example be `Ganga-Ganga-6-1-11`


I think it's worth noting that this function is a general one that will guarantee that you have the old style as long as you strip the `Ganga-` before the replace, such that if you pass it the new `.` style you will get the same as it you pass it the `-` style, namely the old style so it should be safe to use in each case.